tests/Vagrantfile

```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "centos65"
+  config.vm.box_url = "http://downloads.juwai.io/devops/boxes/centos65-x86-64.box"
```

Comment:
Do we have anything private on that box? It seems to be a pretty basic install of centos65.